amf-api-contract/shared/src/main/scala/amf/apicontract/internal/transformation/compatibility/oas3/CleanNullSecurity.scala

```diff
@@ -1,50 +0,0 @@
-package amf.apicontract.internal.transformation.compatibility.oas3
```

Comment:
What is this change?

Reply:
The transformation step was duplicated, once in oas2 and once in oas3, they both did the same, so I deleted the duplicated one.

Reply:
done